composer/algorithms/cutmix/README.md

```diff
@@ -28,13 +29,9 @@ def training_loop(model, train_loader):
 
     for epoch in range(num_epochs):
         for X, y in train_loader:
-            X_cutmix, y_cutmix = cf.cutmix_batch(X,
-                                                 y,
-                                                 num_classes=1000,
-                                                 alpha=1.0)
-
+            X_cutmix, y_perm, area, _ = cutmix_batch(X, y, alpha=0.2)
             y_hat = model(X_cutmix)
-            loss = loss_fn(y_hat, y_cutmix)
+            loss = area * loss_fn(y_hat, y) + (1 - area) * loss_fn(y_hat, y_perm)
```

Comment:
Thanks for updating the readme to reflect the change! Are we okay with adding extra logic for the user (outside of just calling the algorithm's functional implementation)? Cory and Landan, you know better than me: is our philosophy that if you're using functional then you have to do a little extra work around the edges?

Reply:
This is a good question, and I'm a little torn on it. This definitely makes the functional implementation less clean to allow for the functional and composer algorithm implementations to share the same code. The other option here would be to have a totally separate functional implementation that preserves the old behavior. I did things this way to allow for the `interpolate_loss=True` behavior to be replicated functionally if one so chooses, but open to other ideas here too.